redfish-core/lib/log_services.hpp

```diff
@@ -1599,25 +1620,47 @@ inline void requestRoutesDBusEventLogEntryCollection(App& app)
                                     }
                                 }
                             }
+                            else if (interfaceMap.first ==
+                                     "org.open_power.Logging.PEL.Entry")
+                            {
+                                for (auto& propertyMap : interfaceMap.second)
+                                {
+                                    if (propertyMap.first == "Subsystem")
+                                    {
+                                        subsystem = std::get_if<std::string>(
+                                            &propertyMap.second);
+                                    }
+                                }
+                            }
                         }
                         // Object path without the
                         // xyz.openbmc_project.Logging.Entry interface, ignore
                         // and continue.
-                        if (id == nullptr || message == nullptr ||
-                            severity == nullptr)
+                        if (id == nullptr || severity == nullptr)
```

Comment:
Need to check nullptr for eventId and resolution?

Reply:
Please check line 1604, I have both checks there